src/ui/accessibility-and-localization/internationalization.md

```diff
@@ -345,6 +345,11 @@ return MaterialApp(
 [`MaterialApp.onGenerateTitle`]: {{site.api}}/flutter/material/MaterialApp/onGenerateTitle.html
 
 ### Placeholders, plurals, and selects
+{{site.alert.note}}
+  When using VS Code, the [arb-editor](https://marketplace.visualstudio.com/items?itemName=Google.arb-editor)
+  extension can help with editing `.arb` templates file through syntax
+  hightlighting, snippets, diagnostics, and quick fixes.
+{{site.alert.end}}
```

Comment:
**issue:** This needs a blank line before it.
**suggestion:** Write as shorter sentences per [GSG](https://developers.google.com/style/accessibility#ease-of-reading).

```suggestion

{{site.alert.note}}
  When using VS Code, add the [arb-editor extension][].
  This extension adds syntax hightlighting, snippets, diagnostics, and quick fixes.
  to help edit `.arb` template files.
{{site.alert.end}}

[arb-editor extension]: https://marketplace.visualstudio.com/items?itemName=Google.arb-editor
```

Reply:
That sounds much better, thanks!